### ISSUE TITLE
BUG - sev-3 - All - Oracle patients don't have loading or error information about the Cerner locations call

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8863]
## What happened?

We don't currently have an error or loading state for Cerner "Oracle Health" users on any screens - home, health category, any feature screen where we'd show the Cerner disclaimers (like messages, appointments, etc).

So basically, the displays for all three of these states are all the same (they are all absent of content):
* We are still waiting to hear if you're a Cerner patient
* We tried to figure out if you're a Cerner patient but it errored out, we have no idea about your facilities
* We are certain you are NOT a Cerner patient

For.... nearly every other call in the app, we'd have different displays for each of those states (loading, error, and null). I thought seeing the distinction via table kind of helps:
<img width="1001" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2w4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d4fe03547c071028f49749cda6d6348360df3278/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

Should we better differentiate between them by having loading & error indicators for Cerner stuff? I think yes.

But - UX team will need to figure out if they'd like to make a change, and what it would be

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
Have the blocked-UX label because this isn't workable by FE until a decision is made if this is a change we should make, and what the display/content would look like

**Comment by Sparowhawk** [comment]
@ala-yna this one is still blocked as we need some copy and design input

**Comment by mistymg** [comment]
@ala-yna for this discussion with Therese, can you include Lauren? I'd like design input, too.

**Comment by mistymg** [comment]
@TKDickson Just to confirm, it's not that you're saying there are NO loading indicators at all happening on these screens. Just that we don't have a specific loading indicator explicitly calling out that we're checking if they're a OH patient?

**Comment by TKDickson** [comment]
@mistymg that's correct. On the screens where we have Cerner/Oracle-related information (Home, Health Landing, the various feature landing pages), we have loading indicators for _other_ stuff on those screens, and not the Cerner/Oracle stuff.

**Comment by jennb33** [comment]
1/7/2025 update: @nhuckleberry6 will review this ticket to determine what from UX is blocking the ticket. More to come.

**Comment by jennb33** [comment]
1/21/2025 - Pending feedback and information from Ryan about Oracle/Cerner general information (no documentation). Moving to blocked.

**Comment by jennb33** [comment]
1/23/2025 - Rolling this ticket to Sprint 10, as it is still blocked, pending information from Ryan re: Cerner and Oracle patients.

**Comment by Eallan919** [comment]
I removed Natasha from this ticket. She is going to spin up a separate ticket to track the UX work portion. i.e. investigating errors.  Then, once she is done on that end, we can link that to this specific ticket.

**Comment by nhuckleberry6** [comment]
1/31/2025 - Holding on this until we can watch the videos on 2/3 @jennb33 found

**Comment by jennb33** [comment]
2/3/2025 - moving this ticket back to Core Mobile team
